notebooks/Binary Classification with the UCI Credit Card Payment Default Dataset.ipynb

```diff
@@ -0,0 +1,599 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Binary Classification with the UCI Credit Card Payment Default Dataset\n",
+    "_**Mitigating introduced demographic disparities in the UCI default of credit card clients dataset**_"
+   ]
```

Comment:
Let's make the title shorter: "Binary Classification with the UCI Credit Default Dataset". And drop the word "demographic" in the subtitle.